src/javascript/app/pages/trade/purchase.js

```diff
@@ -49,7 +49,7 @@ const Purchase = (() => {
         const button             = CommonFunctions.getElementById('contract_purchase_button');
 
         const error      = details.error;
-        const show_chart = !error && passthrough.duration <= 10 && passthrough.duration_unit === 't' && /^(risefall|higherlower|asian|touchnotouch)$/.test(sessionStorage.formname);
+        const show_chart = !error && passthrough.duration <= 10 && passthrough.duration_unit === 't' && !/^(digits)$/.test(sessionStorage.formname);
```

Comment:
I believe the formname for digits are `overunder|evenodd|matchdiff`, please test it, I don't think it works with `digits`